cluster/hook-template.yaml

```diff
@@ -9,6 +9,7 @@ parameters:
   value: arquillian
 - name: VERSION
   required: true
+  value: latest
```

Comment:
How does the `latest` value help here? What's wrong with particular version?

Reply:
Reading issue comment can shed some light here. Plus this discussion I believe https://github.com/arquillian/ike-prow-plugins/issues/100#issuecomment-384276306

Reply:
thanks for pointers.